README.md

```diff
@@ -110,19 +110,6 @@ While simply implementing this protocol is all you need for your object to be a
 
 So for your analytics provider to be able to handle page views in the Simcoe framework, your provider should implement the `PageViewTracking` protocol. This plug-and-play API allows you to define what you want your provider to handle at a very granular level. If your provider only needs to implement location tracking and nothing else, then you need only implement the `LocationTracking` protocol (as well as the `AnalyticsTracking` protocol, of course); all other protocols are optional. This allows you full customization as to how your objects respond to Simcoe.
 
-Each provider type defined above maps directly to a Simcoe function that will call that method. These are, respectively:
```

Comment:
After a conversation with @MichaelCampbell, we decided to remove this section entirely (although I forget the exact reason now...)